replacer/utils/commandProcessor.mjs

```
@@ -1,18 +1,18 @@
+    let content = fs.readFileSync(filePath, 'utf8')
+
+    patterns.forEach(replace => {
+      content = content
+        .replace(replace.find, replace.replace)
+    })
+    fs.writeFileSync(outputFilePath, content, 'utf8')
+    console.log(`Processed file saved to: ${outputFilePath}`)
+
+    return patterns
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)